.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,42 @@
+<!--
+For urgent operational issues, please contact AWS Support directly.
+https://aws.amazon.com/premiumsupport/
+
+Please provide the following information:
+-->
+
+
+### Summary
+<!-- Please provide a brief outline of the issue -->
+
+
+### Description
+<!-- Provide detailed information about this issue -->
+
+
+
+<!-- Not required for feature requests -->
+### Expected Behavior
+
+
+### Observed Behavior
+
+
+### Environment Details
+<!--
+Examples:
+* docker info
+* curl http://localhost:51678/v1/metadata
+* df -kh
+-->
+
+
+### Supporting Log Snippets
+<!--
+Please have a look at https://github.com/awslabs/ecs-logs-collector
+for the data we typically require for investigation
+
+GitHub issues are public, please remove sensitive data from logs before posting
```

Comment:
I agree here but not all log lines contain sensitive data and for some issues, customers could go ahead and post them using this template. If they are unsure, they should just hold off and request alternative means for sharing the logs. The line added here should serve as a warning. 

If customers are just going to describe the issue without any supporting logs; its going to get extremely difficult to make meaningful inferences.